selectrum.el

```diff
@@ -196,32 +194,22 @@ properties will retain their ordering, which may be significant
 \(e.g. for `load-path' shadows in `read-library-name')."
   :type 'function)
 
-(defun selectrum-default-candidate-highlight-function (input candidates)
-  "Default value of `selectrum-highlight-candidates-function'.
-Highlight the substring match with
-`selectrum-primary-highlight'. INPUT is a string, CANDIDATES is a
-list of strings."
-  (let ((regexp (regexp-quote input)))
-    (save-match-data
-      (mapcar
-       (lambda (candidate)
-         (when (string-match regexp candidate)
-           (setq candidate (copy-sequence candidate))
-           (put-text-property
-            (match-beginning 0) (match-end 0)
-            'face 'selectrum-primary-highlight
-            candidate))
-         candidate)
-       candidates))))
-
-(defcustom selectrum-highlight-candidates-function
-  #'selectrum-default-candidate-highlight-function
+(defun selectrum-ignore-highlighting (_input candidates)
```

Comment:
I would call this rather something like identity.

Reply:
Hm, I was trying to find a name which is clear for the user but ignoring doesn't fit well it rather keeps the highlighting.

Reply:
selectrum-highlight-pass-through, selectrum-highlight-identity, selectrum-candidates-identity, selectrum-candidates-passthrough, something like this?

Reply:
What about selectrum-no-op?

Reply:
I don't like it. My favorite would be selectrum-candidates-identity.

Reply:
The functions are selectrum-refine-candidates-function and selectrum-highlight-candidates-function. And selectrum-candidates-identity would work for both I think. Not really useful, but it would work.

Reply:
Okay, I will got with that